### ISSUE TITLE
I dont have a skin when i play

### ISSUE THREAD

**Issue by N3sher** [AllTheMods/ATM-6#516]
![2020-12-07_22 24 58](https://user-images.githubusercontent.com/75643678/101407830-c0e29480-38db-11eb-9830-3fec40fccc66.png)

**Describe the bug**
When I join a world my skin wont load, but if i press "m" i see my skin on the map.

**To Reproduce**
1. Go to a world

**Expected behavior**
Have a skin.
**Screenshots**

![2020-12-07_22 24 52](https://user-images.githubusercontent.com/75643678/101407650-78c37200-38db-11eb-989f-7b13f0f0ffdf.png)
![2020-12-07_22 19 20](https://user-images.githubusercontent.com/75643678/101407659-7a8d3580-38db-11eb-8d97-e41d813ab90a.png)

**Comment by ViRb3** [comment]
Are you running the server in offline mode?

**Comment by lauash** [comment]
I have the same issue, and it's on a single player world. I'm not in offline mode.

**Comment by thevortex** [comment]
have either of you migrated your mojang account to a microsoft account?

**Comment by lauash** [comment]
I don't believe so, and I don't think I have the ability to yet on this account.

**Comment by lauash** [comment]
Is there anything else that could be causing this? I don't have the issue on a 1.12 modded instance, and with vanilla 1.16. I did notice when I go to the multiplayer screen I have to manually login but doing so has no affect on my skin afterwards.

**Comment by ViRb3** [comment]
I suppose a firewall or other similar filtering program could be blocking the request to Minecraft's skin retrieving server. The logs should note this issue at some point, please check them well. There's also mods like https://www.curseforge.com/minecraft/mc-mods/offlineskins as a workaround.

**Comment by lauash** [comment]
I've tried both GDLauncher and MultiMC. It works with vanilla Minecraft yes, and in a modded 1.12 (same launcher) like I said before. I'm not sure what to look for in the logs, or which logs really look at. I went through the latest after signing in and out immediately and didn't see anything regarding contacting any server or errors that could be related. I turned my firewall completely off at one point to see if it made a difference, it didn't and I turned it back on. Here are the logs from the "latest" file after just opening the game and loading my save: https://paste.ee/p/hDnC1. I'll try the mod and see if that works in the mean time.

**Comment by lauash** [comment]
Alright so as I'm looking into the offline skin mod, I discovered when I look up my UUID on any website it doesn't show my skin. The official Minecraft launcher and website both show them, and as I've said before they work in other instances...so I don't really know what I can do since it seems to be tied to my account. I'll probably open a ticket with Microsoft and see what they say.

**Comment by ViRb3** [comment]
Try changing your skin back and forth to another skin, see if that helps. Are you sure you didn't migrate your account to Microsoft somehow, or buy your account with Microsoft in the first place?

**Comment by lauash** [comment]
I bought the game through Mojang back in Sept. 2019, I just looked for the receipt. And I've never used a Microsoft login and don't have any evidence that my account is even eligible for the migration. For reference I'm able to login with this account on MultiMC and I know that it can't authorize Microsoft accounts yet. I have a second account, that's older than this one, as well and it hasn't worked either. 

The issue with my skin not showing on the websites was only on those 2 (which used the same service to grab the skin). I tried the offline skin mod, and since the online service it uses is the one that wasn't working for me (Crafatar), I did the local file option, and it doesn't work either. I will say I changed my skin through the Minecraft website a 30 min ago and it didn't update the mini-map when I completely restarted the game just now so I'm not sure what that could mean.
